File: Backend/MedicalApp/Models/offer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class offer
    {
        [Key]
        public int id { get; set; }
        [ForeignKey("place")]
        public string place_id { get; set; }
        public int offer_percent { get; set; }
        public string comment { get; set; }
        public string expired_date { get; set; }
        public virtual place place { get; set; }
    }
}

File: Frontend/Graduation Project Frontend/GpFront/src/app/_models/offer.ts
export class Offer {

    constructor(
        public comment ?: string,
        public expired_date ?: string,
        public offer_percent ?: number,
        public place_id ?: string,
        public place_name ?: string,
        public rate ?: number,
        public type ?: number,
        public img_src ?: string,
        public id?:number
    ){}

}
